src/app/shared/components/template/components/latex/latex.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from "@angular/core";
+import { TemplateBaseComponent } from "src/app/shared/components/template/components/base";
+
+@Component({
+  selector: "template-latex-component",
+  template: ` <span [innerHTML]="latexInput | latex"> </span> `,
```

Comment:
It would probably be easier to just bind directly to `_row.value` instead of re-casting to the `latexInput` variable. This also removes any need for `ngOnInit` and `getParams` so actually everything will just be handled by the pipe